build.gradle

```diff
@@ -134,42 +141,28 @@ jar {
 
 jar.dependsOn test
 
-configurations {
-  api
-  cliOutput.extendsFrom (cliCompile)
-  harness
-  translate
-  testsAsJar
-}
-
 task cliJar(type: Jar, dependsOn: cliClasses) {
     baseName = "commcare-cli"
     from sourceSets.translate.output
     from sourceSets.cli.output
     from sourceSets.main.output
     from {
-        configurations.cliCompile.collect {
-            it.isDirectory() ? it : zipTree(it)
-        }
-        configurations.cliRuntime.collect {
+        configurations.cliOutput.collect {
```

Comment:
@ShivamPokhriyal Curious if there is a reason you decided to use `cliOutput` here instead of just `cli` like you did it for `ccapi` below ?

Reply:
I think, I did it because the `configurations` defined it as `cliOutput` instead of `cli` 
https://github.com/dimagi/commcare-core/pull/1034/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7L139

Reply:
Ahh I see. Thanks!